profiles/models.py

```diff
@@ -119,12 +121,14 @@ def save(
             or self.name != self.__previous_name
             or self.headline != self.__previous_headline
```

Comment:
Do we need to add `bio` here too? Is `headline` something which needs to be updated for all posts but `bio` isn't?

Reply:
`author_bio` isn't in the post/comment ES mapping, but `author_name` and `author_headline` are.